lifecycle/distribute.go

```diff
@@ -1,21 +1,19 @@
 package lifecycle
 
 import (
-	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
 	"github.com/jfrog/jfrog-cli-core/v2/common/spec"
 	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
 	"github.com/jfrog/jfrog-client-go/lifecycle/services"
-	"github.com/jfrog/jfrog-client-go/utils/distribution"
 )
 
 type ReleaseBundleDistributeCommand struct {
-	serverDetails           *config.ServerDetails
-	distributeBundlesParams distribution.DistributionParams
-	distributionRules       *spec.DistributionRules
-	dryRun                  bool
-	autoCreateRepo          bool
-	pathMappingPattern      string
-	pathMappingTarget       string
+	releaseBundleCmd
+	distributionRules  *spec.DistributionRules
+	dryRun             bool
+	autoCreateRepo     bool
```

Comment:
It the autoCreateRepo feature supported in v2?

Reply:
[Yep](https://jfrog.com/help/r/jfrog-rest-apis/distribute-release-bundle-version-v2)